web/pgadmin/tools/sqleditor/static/js/components/sections/ResultSet.jsx

```diff
@@ -1046,6 +1047,10 @@ export function ResultSet() {
       setSelectedRows(new Set());
       setAllRowsSelect('NONE');
     });
+
+    eventBus.registerListener(QUERY_TOOL_EVENTS.SET_QUERY_DATA, (data) => {
+      setQueryData(data);
```

Comment:
You can directly call this.

Reply:
Yes, I can call it directly as a prop.
But I thought if in future we can use that event.
Let me know if we don't want this event and I will call it directly.